recipes/latch/meta.yaml

```diff
@@ -1,122 +1,71 @@
-    - pyflyte-fast-execute=flytekit.bin.entrypoint:fast_execute_task_cmd
-    - pyflyte-map-execute=flytekit.bin.entrypoint:map_execute_task_cmd
-    - pyflyte=flytekit.clis.sdk_in_container.pyflyte:main
-    - flyte-cli=flytekit.clis.flyte_cli.main:_flyte_cli
     - latch=latch_cli.main:main
-
+  run_exports:
+    - {{ pin_subpackage('latch', max_pin="x") }}
+    
 package:
   name: {{ name|lower }}
-  version: {{ latch_version }}
+  version: {{ version }}
 
 source:
-  - url: https://files.pythonhosted.org/packages/e1/66/4a57ac840367ad4deeb3725ae5eef7d8dab957e34eb570e838c4b6f6236b/latch-{{ latch_version }}.tar.gz
-    sha256: 175a07218a2f9f8c90b61be1e25e2ecc92750b4e41c3d3b9f47de96d2fb57090
-    folder: latch
-  - url: https://files.pythonhosted.org/packages/f9/22/dbbd7ecff5ba620f94c6a4b889ed4bd9caa0a994847cdb717266094ce28c/lyteidl-{{ lyteidl_version }}.tar.gz
-    sha256: ffd621d17542dc4aaf954043010aaff32da2733218d991d1959d907b90c7413c
-    folder: lyteidl
-  - url: https://files.pythonhosted.org/packages/ad/a9/ceb1f09547178b7b8c34eca11038d6a5b50cb507b96627806835b0159ea1/lytekit-{{ lytekit_version }}.tar.gz
-    sha256: f1f114bd18fda5e7ffb8210adabe7caff9acb68accff087cd4847ac043b65150
-    folder: lytekit
-  - url: https://files.pythonhosted.org/packages/3e/18/54873b40638d9f5dbbac624cf1d7d8001490646cde818604f337f65c0380/lytekitplugins-pods-{{ lytekitplugins_pods_version }}.tar.gz
-    sha256: 6fecca449e49824e41ad0f6dc7946baa1fefe20b4d59eb1b59882601df09c6f6
-    folder: lytekitplugins-pods
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/latch-{{ version }}.tar.gz
+  sha256: 56db74621fc311e07d504e26442d8003247b8adc24b365eff9bb179281b5b7d8
+  patches:
+    - 00001.patch
 
 requirements:
   host:
-    - python >=3.8
+    - python >=3.9,<3.13
+    - hatchling >=1.27.0
+    - pip
   run:
-    - aioconsole ==0.5.1
-    - apscheduler ==3.9.1
-    - asyncssh ==2.12.0
-    - awscli ==1.25.22
-    - boto3 >=1.24.22
-    - click >=8.0,<9.0
-    - cloudpickle >=2.0.0
-    - cookiecutter >=1.7.3
-    - croniter >=0.3.20,<4.0.0
-    - dataclasses-json >=0.5.2
-    - deprecated >=1.0,<2.0
-    - diskcache >=5.2.1
-    - docker-image-py >=0.1.10
-    - docker-py >=5.0.3,<6.0.0
-    - docstring_parser >=0.9.0
-    - googleapis-common-protos
-    - gql ==3.4.0
+    - python >=3.9
+    - aioconsole ==0.6.1
+    - apscheduler >=3.10.0
+    - asyncssh ==2.13.2
+    - boto3 >=1.26.0
+    - click >=8.0
+    - docker-py >=7.1.0
+    - gitpython ==3.1.40
+    - gql ==3.5.0
     - graphql-core ==3.2.3
-    - grpcio >=1.43.0,!=1.45.0,<2.0
-    - grpcio-status >=1.43,!=1.45.0
-    - jsonschema >=4.5.1
-    - keyring >=18.0.1
-    - marshmallow-jsonschema >=0.12.0
-    - natsort >=7.0.1
-    - numpy <1.22.0
-    - pandas >=1.0.0,<2.0.0
-    - paramiko >=2.11.0
-    - prompt-toolkit ==3.0.33
-    - protobuf >=3.6.1,<4.0.0
-    - protoc-gen-swagger
-    - pyarrow >=4.0.0,<7.0.0
+    - latch-persistence >=0.1.5
+    - lytekit ==0.15.28
```

Comment:
Do we really need those strict pins?

Reply:
Generated via `grayskull`, so I didn't ask questions ;).  But probably, given some of the issues I had with minor versions already.